compilers/concrete-compiler/compiler/include/concretelang/ServerLib/ServerLib.h

```diff
@@ -47,7 +47,7 @@ class ServerCircuit {
 
 public:
   /// Call the circuit with public arguments.
-  Result<std::vector<TransportValue>> call(const ServerKeyset &serverKeyset,
```

Comment:
Why is it needed to remove the const ? Are we changing things in the server keyset ?

Reply:
As discussed, for seeded keys to be decompressed.